src/containers/Teams/CreateTeam/components/InviteTeam/index.js

```diff
@@ -1,56 +1,42 @@
-import React, { useCallback, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import _debounce from 'lodash/debounce';
-import _sortBy from 'lodash/sortBy';
-import Swal from 'sweetalert2';
+import { connect } from 'react-redux';
 
 import { FadeDiv } from 'utils';
+import { inviteTeamMembersAction } from 'store/actions/team';
 import MemberItem from './MemberItem';
+import InviteDialog from '../../../TeamMemberView/InviteDialog';
```

Comment:
can we use absolute paths

Reply:
I don't think we need it
But maybe we can move that to the components folder
What's your opinion about that?

Reply:
yes, its good  if we  follow uniformity on all imports. :)